docs/reference/predicates.md

```diff
@@ -152,6 +152,28 @@ route1: Path("/test") && Weight(100) -> "http://www.zalando.de";
 route2: Path("/test") && True() && True() -> "http://www.zalando.de";
 ```
 
+## True
+
+Does always match. Before `Weight` predicate existed this was used to give a route more weight.
+
+Example where `route2` is has more weight.
```

Comment:
"is has" -> "has"

Reply:
Oops, fixed. Please use Github suggestions next time to 🙏